react_practice/react-apps/waitlist-page-v1/src/PrimaryBtn.css

```diff
@@ -1,21 +1,18 @@
-.primary-btn-title {
+.primary-btn {
+  border: 2px solid #1e90ff;
```

Comment:
With the change in the elements used to render the `PrimaryBtn`, I found that the button would change size when I hovered. This was likely due to the added border on the `:hover` state, which adds to the size of the button. Rather than trying to adjust the padding to account for the new border, instead I opted to add a border that matches the size in both states and color it the same as the background. This means I was also able to override the `border-color` only in the hover state.